chapter_computer-vision/anchor.md

```diff
@@ -36,9 +47,46 @@ Y = npx.multibox_prior(X, sizes=[0.75, 0.5, 0.25], ratios=[1, 2, 0.5])
 Y.shape
 ```
 
+```{.python .input}
+#@tab pytorch
+#@save
+def multibox_prior(feature_map_sizes, sizes, aspect_ratios):
```

Comment:
Here it is right that you define this function because torch/torchvision don't offer a built-in method for multibox_prior like mxnet.multibox prior